client/src/www/app/outline_server_repository/access_key_serialization.ts

```diff
@@ -1,121 +0,0 @@
-// Copyright 2018 The Outline Authors
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//      http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-import {SHADOWSOCKS_URI} from 'ShadowsocksConfig';
-
-import {ShadowsocksSessionConfig} from './vpn';
-import * as errors from '../../model/errors';
-
-// DON'T use these methods outside of this folder!
-
-// Parses an access key string into a ShadowsocksConfig object.
-export function staticKeyToShadowsocksSessionConfig(
-  staticKey: string
-): ShadowsocksSessionConfig {
-  try {
-    const config = SHADOWSOCKS_URI.parse(staticKey);
-    return {
-      host: config.host.data,
-      port: config.port.data,
-      method: config.method.data,
-      password: config.password.data,
-      prefix: config.extra?.['prefix'],
-    };
-  } catch (cause) {
-    throw new errors.ServerAccessKeyInvalid('Invalid static access key.', {
-      cause,
-    });
-  }
-}
-
-function parseShadowsocksSessionConfigJson(
-  responseBody: string
-): ShadowsocksSessionConfig | null {
-  const responseJson = JSON.parse(responseBody);
-
-  if ('error' in responseJson) {
-    throw new errors.SessionProviderError(
-      responseJson.error.message,
-      responseJson.error.details
-    );
-  }
-
-  const {method, password, server, server_port, prefix} = responseJson;
-
-  // These are the mandatory keys.
-  const missingKeys = [];
-
-  for (const [key, value] of Object.entries({
-    method,
-    password,
-    server,
-    server_port,
-  })) {
-    if (typeof value === 'undefined') {
-      missingKeys.push(key);
-    }
-  }
-
-  if (missingKeys.length > 0) {
-    throw new TypeError(`Missing JSON fields: ${missingKeys.join(', ')}.`);
-  }
-
-  return {
-    method,
-    password,
-    host: server,
-    port: server_port,
```

Comment:
Here are the code that converts `server`+`server_port` to `host`+`port`. This PR removed this logic.